doc/en/user/source/services/wfs/axis_order.rst

```diff
@@ -0,0 +1,676 @@
+.. _wfs_basics_axis:
+
+Axis ordering
+=============
+
+WFS 1.0.0 servers return geographic coordinates in longitude/latitude (x/y) order, the most common way to distribute data. For example, many shapefiles adopt this order by default. 
```

Comment:
Simply talking about longitude and latitude is limiting and gives false impressions. There is a number of projected CRSs that are north/east oriented in their EPSG definition, those are affected by the axis order rules too. Would be better to talk about "respecting EPSG axis order" vs "forcing east/north" order. It is also to be noted that there are CRSs, like the polar stereographic based ones, where talking about east and north makes no sense, having a pole in the middle of the CRS axis.